### ISSUE TITLE
Path of datasets

### ISSUE THREAD

**Issue by va1bhv** [Xotik105/ML_projects#1]
# Path of datasets
I found the following interesting notes regarding the files:  
* The path of the datasets is absolute in all the files. If anyone clones the repository, they would have to manually change the paths before any of the files could be run. Relative naming of the paths for the datasets would probably be the best course of action.
* Adding a requirements.txt will also be very informative as it would make it clear what are the dependencies for this repository as well as a quick way to install the dependencies.
* Although the dataset is very simple and self-explanatory, some metadata about it would make the data understanding process much easier.

**Comment by Xotik105** [comment]
Yes! this repo is still in the building, accumulating all the projects for now.  I wont advice to clone this. This repo is suited for my personal use and tracker of everything that i did and will do. Thank you for your time and your points are noted.
